tools/scaffold/scripts/schema-scaffold.sh

```
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+
```

the script should fail here in case `name` is not defined.

---

makes sense, adjusted